translate.js

```diff
@@ -0,0 +1,8 @@
+//modules
+const { getWordInformation, translate } = require('./index.js')
+
+//sentence
+var sentence = 'Gallia est omnis divisa in partes tres, quarum unam incolunt Belgae, aliam Aquitani, tertiam qui ipsorum lingua Celtae, nostra Galli appellantur.'
```

Comment:
here's also a `var` (but this does not count, as this is not part of the package